lib/ui/platform_dispatcher.dart

```diff
@@ -7,6 +7,14 @@ part of dart.ui;
 typedef VoidCallback = void Function();
 
 /// Signature for [PlatformDispatcher.onBeginFrame].
+///
+/// The `duration` argument is the point at which the current frame interval
```

Comment:
FYI @chinmaygarde

Reply:
chinmay is out right now. is this blocking?